pkg/security/probe/model.go

```diff
@@ -120,19 +133,38 @@ type FileEvent struct {
 	PathnameStr     string `field:"filename" handler:"ResolveInode,string"`
 	ContainerPath   string `field:"container_path" handler:"ResolveContainerPath,string"`
 	BasenameStr     string `field:"basename" handler:"ResolveBasename,string"`
+
+	pathResolutionError error `field:"-"`
```

Comment:
nit: I think the tag field is useless here as the field is not exported

Reply:
👍🏻

Reply:
I get a `invalid memory address or nil pointer dereference` error when I run go generate without it :(